intgtest/intgSpec.js

```diff
@@ -267,7 +267,7 @@ describe('Vantiq SDK Integration Tests', function() {
         };
 
         // Insert message
-        return v.publish('topics', '/test/topic', message)
+        return v.publish('topics', '/test/topic3', message)
```

Comment:
is the '3' really supposed to be there, or is this a typo?
I don't see a similar change to the rule _onTestPublish.vail_